yaksh/templates/exam.html

```diff
@@ -88,17 +89,39 @@
       <div class="panel-heading">Testcase No. {{ forloop.counter }}</div>
```

Comment:
I feel instead of TestCase No we can name it to Error No. Since the errors we get are not specific to test case every time.

Reply:
resolved